.buckconfig

```diff
@@ -0,0 +1,6 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
```

Comment:
why the target is 23 which is android 6 ?

Reply:
it's default value, actual value will be defined based on the latest supported version of android provided by bs/saucelabs. Will be changed later